booking-engine-api/operations/availability-blocks.md

```diff
@@ -0,0 +1,66 @@
+# Availability blocks
+
+## Get Availability blocks
+
+Get availability block details for specified booking engine instance, that modifies your availability of space categories, and rates.
```

Comment:
The description should be something like "Get details for the given availability blocks for the given hotel or enterprise".